### PR TITLE
Add spack-stack/1.6.0 support

### PR DESCRIPTION
**Pull request by DavidHuber-NOAA**
This upgrades GDASApp libraries to spack-stack version 1.6.0.  Most notably, this upgrades the following libraries

netcdf-fortran 4.6.0 -> 4.6.1
boost 1.78 -> 1.83.0
openblas 0.3.19 -> 0.3.24
sp 2.3.3 -> 2.5.0

Regression tests were run on Hera by @RussTreadon-NOAA and @CoryMartin-NOAA.

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
All credit goes to @RussTreadon-NOAA for testing! Thanks @DavidHuber-NOAA for this PR.

**Review by CoryMartin-NOAA — DISMISSED** [review]

**Comment by RussTreadon-NOAA** [comment]
**Orion test**
Install `DavidHuber-NOAA:feature/ss160` on Orion inside g-w `develop` @ [`ee6f536`](https://github.com/NOAA-EMC/global-workflow/commit/ee6f536ea0228c60f5a8bec4037cd6f7ea63b816).  Execute g-w `build_all.sh -u`.  GDASApp built using spack-stack/1.6.0.  Run ctests.  All 52 tests pass on Orion
```
/work/noaa/da/rtreadon/git/global_workflow/spackstack160_orion/sorc/gdas.cd/build$ test -R test_gdasapp
Test project /work/noaa/da/rtreadon/git/global_workflow/spackstack160_orion/sorc/gdas.cd/build
      Start 1340: test_gdasapp_util_coding_norms
 1/52 Test #1340: test_gdasapp_util_coding_norms ........................   Passed    1.19 sec
      Start 1341: test_gdasapp_util_ioda_example
 2/52 Test #1341: test_gdasapp_util_ioda_example ........................   Passed    2.20 sec
      Start 1342: test_gdasapp_util_prepdata
 3/52 Test #1342: test_gdasapp_util_prepdata ............................   Passed    0.59 sec
      Start 1343: test_gdasapp_util_rads2ioda
 4/52 Test #1343: test_gdasapp_util_rads2ioda ...........................   Passed    0.48 sec
      Start 1344: test_gdasapp_util_ghrsst2ioda
 5/52 Test #1344: test_gdasapp_util_ghrsst2ioda .........................   Passed    0.14 sec
      Start 1345: test_gdasapp_util_smap2ioda
 6/52 Test #1345: test_gdasapp_util_smap2ioda ...........................   Passed    0.14 sec
      Start 1346: test_gdasapp_util_smos2ioda
 7/52 Test #1346: test_gdasapp_util_smos2ioda ...........................   Passed    0.15 sec
      Start 1347: test_gdasapp_util_viirsaod2ioda
 8/52 Test #1347: test_gdasapp_util_viirsaod2ioda .......................   Passed    0.17 sec
      Start 1348: test_gdasapp_util_icecamsr2ioda
 9/52 Test #1348: test_gdasapp_util_icecamsr2ioda .......................   Passed    0.14 sec
      Start 1680: test_gdasapp_check_python_norms
10/52 Test #1680: test_gdasapp_check_python_norms .......................   Passed    1.32 sec
      Start 1681: test_gdasapp_check_yaml_keys
11/52 Test #1681: test_gdasapp_check_yaml_keys ..........................   Passed    0.09 sec
      Start 1682: test_gdasapp_jedi_increment_to_fv3
12/52 Test #1682: test_gdasapp_jedi_increment_to_fv3 ....................   Passed    0.43 sec
      Start 1683: test_gdasapp_convert_ewok_yaml
13/52 Test #1683: test_gdasapp_convert_ewok_yaml ........................   Passed    0.18 sec
      Start 1684: test_gdasapp_setup_cycled_exp
14/52 Test #1684: test_gdasapp_setup_cycled_exp .........................   Passed    0.82 sec
      Start 1685: test_gdasapp_convert_bufr_temp_dbuoy
15/52 Test #1685: test_gdasapp_convert_bufr_temp_dbuoy ..................   Passed    0.24 sec
      Start 1686: test_gdasapp_convert_bufr_salt_dbuoy
16/52 Test #1686: test_gdasapp_convert_bufr_salt_dbuoy ..................   Passed    0.22 sec
      Start 1687: test_gdasapp_convert_bufr_temp_mbuoyb
17/52 Test #1687: test_gdasapp_convert_bufr_temp_mbuoyb .................   Passed    0.22 sec
      Start 1688: test_gdasapp_convert_bufr_salt_mbuoyb
18/52 Test #1688: test_gdasapp_convert_bufr_salt_mbuoyb .................   Passed    0.22 sec
      Start 1689: test_gdasapp_convert_bufr_tesacprof
19/52 Test #1689: test_gdasapp_convert_bufr_tesacprof ...................   Passed    0.26 sec
      Start 1690: test_gdasapp_convert_bufr_trkobprof
20/52 Test #1690: test_gdasapp_convert_bufr_trkobprof ...................   Passed    0.27 sec
      Start 1691: test_gdasapp_convert_bufr_sfcships
21/52 Test #1691: test_gdasapp_convert_bufr_sfcships ....................   Passed    0.22 sec
      Start 1692: test_gdasapp_convert_bufr_sfcshipsu
22/52 Test #1692: test_gdasapp_convert_bufr_sfcshipsu ...................   Passed    0.23 sec
      Start 1693: test_gdasapp_soca_nsst_increment_to_mom6
23/52 Test #1693: test_gdasapp_soca_nsst_increment_to_mom6 ..............   Passed    1.20 sec
      Start 1694: test_gdasapp_soca_prep
24/52 Test #1694: test_gdasapp_soca_prep ................................   Passed    3.34 sec
      Start 1695: test_gdasapp_soca_run_clean
25/52 Test #1695: test_gdasapp_soca_run_clean ...........................   Passed    0.48 sec
      Start 1696: test_gdasapp_soca_setup_obsprep
26/52 Test #1696: test_gdasapp_soca_setup_obsprep .......................   Passed    0.97 sec
      Start 1697: test_gdasapp_soca_JGLOBAL_PREP_OCEAN_OBS
27/52 Test #1697: test_gdasapp_soca_JGLOBAL_PREP_OCEAN_OBS ..............   Passed   43.60 sec
      Start 1698: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP
28/52 Test #1698: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP ....   Passed   42.24 sec
      Start 1699: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT
29/52 Test #1699: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT ....   Passed   74.24 sec
      Start 1700: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN
30/52 Test #1700: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN .....   Passed   42.23 sec
      Start 1701: test_gdasapp_soca_copy_scratch
31/52 Test #1701: test_gdasapp_soca_copy_scratch ........................   Passed    0.71 sec
      Start 1702: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_CHKPT
32/52 Test #1702: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_CHKPT ...   Passed   42.22 sec
      Start 1703: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_POST
33/52 Test #1703: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_POST ....   Passed  106.27 sec
      Start 1704: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY
34/52 Test #1704: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY ....   Passed  362.32 sec
      Start 1705: test_gdasapp_soca_socahybridweights
35/52 Test #1705: test_gdasapp_soca_socahybridweights ...................   Passed  202.29 sec
      Start 1706: test_gdasapp_soca_incr_handler
36/52 Test #1706: test_gdasapp_soca_incr_handler ........................   Passed   74.22 sec
      Start 1707: test_gdasapp_soca_ens_handler
37/52 Test #1707: test_gdasapp_soca_ens_handler .........................   Passed   10.23 sec
      Start 1708: test_gdasapp_land_create_ens
38/52 Test #1708: test_gdasapp_land_create_ens ..........................   Passed    0.65 sec
      Start 1709: test_gdasapp_land_imsproc
39/52 Test #1709: test_gdasapp_land_imsproc .............................   Passed    2.73 sec
      Start 1710: test_gdasapp_land_apply_jediincr
40/52 Test #1710: test_gdasapp_land_apply_jediincr ......................   Passed    7.26 sec
      Start 1711: test_gdasapp_land_letkfoi_snowda
41/52 Test #1711: test_gdasapp_land_letkfoi_snowda ......................   Passed   22.04 sec
      Start 1712: test_gdasapp_convert_bufr_adpsfc_snow
42/52 Test #1712: test_gdasapp_convert_bufr_adpsfc_snow .................   Passed    2.87 sec
      Start 1713: test_gdasapp_convert_bufr_adpsfc
43/52 Test #1713: test_gdasapp_convert_bufr_adpsfc ......................   Passed    3.91 sec
      Start 1714: test_gdasapp_convert_gsi_satbias
44/52 Test #1714: test_gdasapp_convert_gsi_satbias ......................   Passed    1.43 sec
      Start 1715: test_gdasapp_setup_atm_cycled_exp
45/52 Test #1715: test_gdasapp_setup_atm_cycled_exp .....................   Passed    0.88 sec
      Start 1716: test_gdasapp_atm_jjob_var_init
46/52 Test #1716: test_gdasapp_atm_jjob_var_init ........................   Passed   44.27 sec
      Start 1717: test_gdasapp_atm_jjob_var_run
47/52 Test #1717: test_gdasapp_atm_jjob_var_run .........................   Passed  106.77 sec
      Start 1718: test_gdasapp_atm_jjob_var_final
48/52 Test #1718: test_gdasapp_atm_jjob_var_final .......................   Passed   42.66 sec
      Start 1719: test_gdasapp_atm_jjob_ens_init
49/52 Test #1719: test_gdasapp_atm_jjob_ens_init ........................   Passed  300.38 sec
      Start 1720: test_gdasapp_atm_jjob_ens_run
50/52 Test #1720: test_gdasapp_atm_jjob_ens_run .........................   Passed  619.75 sec
      Start 1721: test_gdasapp_atm_jjob_ens_final
51/52 Test #1721: test_gdasapp_atm_jjob_ens_final .......................   Passed  266.61 sec
      Start 1722: test_gdasapp_aero_gen_3dvar_yaml
52/52 Test #1722: test_gdasapp_aero_gen_3dvar_yaml ......................   Passed    0.62 sec

100% tests passed, 0 tests failed out of 52

Label Time Summary:
gdas-utils    =   5.21 sec*proc (9 tests)
script        =   5.21 sec*proc (9 tests)

Total Test time (real) = 2440.01 sec
```

**Comment by RussTreadon-NOAA** [comment]
**Hercules test**

Install `DavidHuber-NOAA:feature/ss160` on Hercules inside g-w develop @ [`ee6f536`](https://github.com/NOAA-EMC/global-workflow/commit/ee6f536ea0228c60f5a8bec4037cd6f7ea63b816). Execute g-w `build_all.sh -u`. GDASApp successfully built on Hercules using spack-stack/1.6.0.  Execute g-w `link_workflow.sh`.  Execute `test_gdasapp` ctests.   Multiple tests fail.

Examine log files for failed jobs.  Find familiar error message
```
    from numpy.core._multiarray_umath import (
ModuleNotFoundError: No module named 'numpy.core._multiarray_umath'
```
Check loaded modules.   See that `py-numpy/1.22.3` is loaded.   Check `hercules.lua`.  See that several modules are reloaded after gdasapp

```
load("hpc/1.2.0")
unload("python/3.10.13")
unload("py-numpy/1.22.3")
load("miniconda3/4.6.14")
load("gdasapp/1.0.0")

load("boost/1.83.0")
load("bufr/12.0.1")
load("fckit/0.11.0")
load("atlas/0.35.1")
load("py-pybind11/2.11.0")
```
This is done in the original `hercules.lua`.   I can't recall why it's done.  Remove the second set of loads for boost, bufr, fckit, atlas, and py-pybind11.   Rebuild `gdas.cd` and rerun test_gdasapp ctests.   This time 52 of 52 tests pass
```
(gdasapp) hercules-login-2:/work/noaa/da/rtreadon/git/global_workflow/spackstack160_hercules/sorc/gdas.cd/build$ ctest -R test_gdasapp
Test project /work/noaa/da/rtreadon/git/global_workflow/spackstack160_hercules/sorc/gdas.cd/build
      Start 1340: test_gdasapp_util_coding_norms
 1/52 Test #1340: test_gdasapp_util_coding_norms ........................   Passed    4.12 sec
      Start 1341: test_gdasapp_util_ioda_example
 2/52 Test #1341: test_gdasapp_util_ioda_example ........................   Passed    0.12 sec
      Start 1342: test_gdasapp_util_prepdata
 3/52 Test #1342: test_gdasapp_util_prepdata ............................   Passed    1.88 sec
      Start 1343: test_gdasapp_util_rads2ioda
 4/52 Test #1343: test_gdasapp_util_rads2ioda ...........................   Passed    0.14 sec
      Start 1344: test_gdasapp_util_ghrsst2ioda
 5/52 Test #1344: test_gdasapp_util_ghrsst2ioda .........................   Passed    0.08 sec
      Start 1345: test_gdasapp_util_smap2ioda
 6/52 Test #1345: test_gdasapp_util_smap2ioda ...........................   Passed    0.08 sec
      Start 1346: test_gdasapp_util_smos2ioda
 7/52 Test #1346: test_gdasapp_util_smos2ioda ...........................   Passed    0.08 sec
      Start 1347: test_gdasapp_util_viirsaod2ioda
 8/52 Test #1347: test_gdasapp_util_viirsaod2ioda .......................   Passed    0.08 sec
      Start 1348: test_gdasapp_util_icecamsr2ioda
 9/52 Test #1348: test_gdasapp_util_icecamsr2ioda .......................   Passed    0.08 sec
      Start 1680: test_gdasapp_check_python_norms
10/52 Test #1680: test_gdasapp_check_python_norms .......................   Passed    1.40 sec
      Start 1681: test_gdasapp_check_yaml_keys
11/52 Test #1681: test_gdasapp_check_yaml_keys ..........................   Passed    0.35 sec
      Start 1682: test_gdasapp_jedi_increment_to_fv3
12/52 Test #1682: test_gdasapp_jedi_increment_to_fv3 ....................   Passed    3.17 sec
      Start 1683: test_gdasapp_convert_ewok_yaml
13/52 Test #1683: test_gdasapp_convert_ewok_yaml ........................   Passed    0.22 sec
      Start 1684: test_gdasapp_setup_cycled_exp
14/52 Test #1684: test_gdasapp_setup_cycled_exp .........................   Passed    1.38 sec
      Start 1685: test_gdasapp_convert_bufr_temp_dbuoy
15/52 Test #1685: test_gdasapp_convert_bufr_temp_dbuoy ..................   Passed    0.24 sec
      Start 1686: test_gdasapp_convert_bufr_salt_dbuoy
16/52 Test #1686: test_gdasapp_convert_bufr_salt_dbuoy ..................   Passed    0.21 sec
      Start 1687: test_gdasapp_convert_bufr_temp_mbuoyb
17/52 Test #1687: test_gdasapp_convert_bufr_temp_mbuoyb .................   Passed    0.22 sec
      Start 1688: test_gdasapp_convert_bufr_salt_mbuoyb
18/52 Test #1688: test_gdasapp_convert_bufr_salt_mbuoyb .................   Passed    0.22 sec
      Start 1689: test_gdasapp_convert_bufr_tesacprof
19/52 Test #1689: test_gdasapp_convert_bufr_tesacprof ...................   Passed    0.23 sec
      Start 1690: test_gdasapp_convert_bufr_trkobprof
20/52 Test #1690: test_gdasapp_convert_bufr_trkobprof ...................   Passed    0.22 sec
      Start 1691: test_gdasapp_convert_bufr_sfcships
21/52 Test #1691: test_gdasapp_convert_bufr_sfcships ....................   Passed    0.22 sec
      Start 1692: test_gdasapp_convert_bufr_sfcshipsu
22/52 Test #1692: test_gdasapp_convert_bufr_sfcshipsu ...................   Passed    0.21 sec
      Start 1693: test_gdasapp_soca_nsst_increment_to_mom6
23/52 Test #1693: test_gdasapp_soca_nsst_increment_to_mom6 ..............   Passed   39.34 sec
      Start 1694: test_gdasapp_soca_prep
24/52 Test #1694: test_gdasapp_soca_prep ................................   Passed    2.32 sec
      Start 1695: test_gdasapp_soca_run_clean
25/52 Test #1695: test_gdasapp_soca_run_clean ...........................   Passed    0.02 sec
      Start 1696: test_gdasapp_soca_setup_obsprep
26/52 Test #1696: test_gdasapp_soca_setup_obsprep .......................   Passed    0.62 sec
      Start 1697: test_gdasapp_soca_JGLOBAL_PREP_OCEAN_OBS
27/52 Test #1697: test_gdasapp_soca_JGLOBAL_PREP_OCEAN_OBS ..............   Passed   43.58 sec
      Start 1698: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP
28/52 Test #1698: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP ....   Passed   74.23 sec
      Start 1699: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT
29/52 Test #1699: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT ....   Passed   74.25 sec
      Start 1700: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN
30/52 Test #1700: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN .....   Passed   42.24 sec
      Start 1701: test_gdasapp_soca_copy_scratch
31/52 Test #1701: test_gdasapp_soca_copy_scratch ........................   Passed    1.28 sec
      Start 1702: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_CHKPT
32/52 Test #1702: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_CHKPT ...   Passed   42.20 sec
      Start 1703: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_POST
33/52 Test #1703: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_POST ....   Passed   42.23 sec
      Start 1704: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY
34/52 Test #1704: test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY ....   Passed  394.29 sec
      Start 1705: test_gdasapp_soca_socahybridweights
35/52 Test #1705: test_gdasapp_soca_socahybridweights ...................   Passed   42.21 sec
      Start 1706: test_gdasapp_soca_incr_handler
36/52 Test #1706: test_gdasapp_soca_incr_handler ........................   Passed    2.16 sec
      Start 1707: test_gdasapp_soca_ens_handler
37/52 Test #1707: test_gdasapp_soca_ens_handler .........................   Passed   10.16 sec
      Start 1708: test_gdasapp_land_create_ens
38/52 Test #1708: test_gdasapp_land_create_ens ..........................   Passed    1.26 sec
      Start 1709: test_gdasapp_land_imsproc
39/52 Test #1709: test_gdasapp_land_imsproc .............................   Passed    7.38 sec
      Start 1710: test_gdasapp_land_apply_jediincr
40/52 Test #1710: test_gdasapp_land_apply_jediincr ......................   Passed    2.78 sec
      Start 1711: test_gdasapp_land_letkfoi_snowda
41/52 Test #1711: test_gdasapp_land_letkfoi_snowda ......................   Passed    8.10 sec
      Start 1712: test_gdasapp_convert_bufr_adpsfc_snow
42/52 Test #1712: test_gdasapp_convert_bufr_adpsfc_snow .................   Passed    2.27 sec
      Start 1713: test_gdasapp_convert_bufr_adpsfc
43/52 Test #1713: test_gdasapp_convert_bufr_adpsfc ......................   Passed    2.83 sec
      Start 1714: test_gdasapp_convert_gsi_satbias
44/52 Test #1714: test_gdasapp_convert_gsi_satbias ......................   Passed    3.27 sec
      Start 1715: test_gdasapp_setup_atm_cycled_exp
45/52 Test #1715: test_gdasapp_setup_atm_cycled_exp .....................   Passed    1.91 sec
      Start 1716: test_gdasapp_atm_jjob_var_init
46/52 Test #1716: test_gdasapp_atm_jjob_var_init ........................   Passed   46.38 sec
      Start 1717: test_gdasapp_atm_jjob_var_run
47/52 Test #1717: test_gdasapp_atm_jjob_var_run .........................   Passed   74.25 sec
      Start 1718: test_gdasapp_atm_jjob_var_final
48/52 Test #1718: test_gdasapp_atm_jjob_var_final .......................   Passed   74.25 sec
      Start 1719: test_gdasapp_atm_jjob_ens_init
49/52 Test #1719: test_gdasapp_atm_jjob_ens_init ........................   Passed   46.08 sec
      Start 1720: test_gdasapp_atm_jjob_ens_run
50/52 Test #1720: test_gdasapp_atm_jjob_ens_run .........................   Passed  298.33 sec
      Start 1721: test_gdasapp_atm_jjob_ens_final
51/52 Test #1721: test_gdasapp_atm_jjob_ens_final .......................   Passed   42.40 sec
      Start 1722: test_gdasapp_aero_gen_3dvar_yaml
52/52 Test #1722: test_gdasapp_aero_gen_3dvar_yaml ......................   Passed    0.38 sec

100% tests passed, 0 tests failed out of 52

Label Time Summary:
gdas-utils    =   6.65 sec*proc (9 tests)
script        =   6.65 sec*proc (9 tests)

Total Test time (real) = 1438.42 sec
```

@DavidHuber-NOAA , my modified `hercules.lua` is in `/work/noaa/da/rtreadon/git/global_workflow/spackstack160_hercules/sorc/gdas.cd/modulefiles/GDAS`.  Below is a git diff
```
@@ -68,12 +68,6 @@ unload("py-numpy/1.22.3")
 load("miniconda3/4.6.14")
 load("gdasapp/1.0.0")

-load("boost/1.83.0")
-load("bufr/12.0.1")
-load("fckit/0.11.0")
-load("atlas/0.35.1")
-load("py-pybind11/2.11.0")
-
 -- below is a hack because of cmake finding the wrong python...
 setenv("CONDA_PREFIX", "/work2/noaa/da/python/opt/core/miniconda3/4.6.14/envs/gdasapp/")

@@ -85,7 +79,6 @@ local mpinproc = '-n'
 setenv('MPIEXEC_EXEC', mpiexec)
 setenv('MPIEXEC_NPROC', mpinproc)

-setenv('R2D2_CONFIG', '/work2/noaa/da/cmartin/GDASApp/R2D2_SHARED/config_orion.yaml')
 setenv("CRTM_FIX","/work2/noaa/da/cmartin/GDASApp/fix/crtm/2.4.0")
 setenv("GDASAPP_TESTDATA","/work2/noaa/da/cmartin/CI/GDASApp/data")
 prepend_path("PATH","/apps/contrib/NCEP/libs/hpc-stack/intel-2018.4/prod_util/1.2.2/bin")
```

We don't need `R2D2_CONFIG` anymore.

File: modulefiles/GDAS/hercules.lua [hunk 1]
```diff
@@ -15,64 +15,64 @@ load("stack-intel-oneapi-mpi/2021.9.0")
...
 load("hpc/1.2.0")
-unload("python/3.10.8")
-unload("python/3.9.2")
+unload("python/3.10.13")
+unload("py-numpy/1.22.3")
 load("miniconda3/4.6.14")
 load("gdasapp/1.0.0")
 
-load("boost/1.78.0")
+load("boost/1.83.0")
```
Comment (RussTreadon-NOAA) [inline modulefiles/GDAS/hercules.lua]: Tests on hercules find that lines 71 to 75 are not needed.  These lines should be removed.

**Review by RussTreadon-NOAA — DISMISSED** [review]
`hera.lua` and `orion.lua` work as is.  `hercules.lua` requires minor changes:  (1) delete indicated loads, remove `R2D2` line.

I'll approve once `hercules.lua` is updated.

File: modulefiles/GDAS/hercules.lua (hunk 1 above)
Comment (DavidHuber-NOAA) [reply modulefiles/GDAS/hercules.lua]: Done.

**Comment by RussTreadon-NOAA** [comment]
Thank you @DavidHuber-NOAA for updating `hercules.lua`.  I updated my working copy of your branch.  I'll recompile and rerun ctests on Hercules.  All tests should pass.  I'll approve once I get this confirmation.

**Review by RussTreadon-NOAA — APPROVED** [review]
Rebuilt `DavidHuber-NOAA:feature/ss160` at [`a83936b`](https://github.com/DavidHuber-NOAA/GDASApp/commit/a83936b1bb6455d58cd7c80bbc1590cd646719c9) on Hercules within g-w.  Run ctests.  All tests pass.

```
(gdasapp) hercules-login-3:/work/noaa/da/rtreadon/git/global_workflow/spackstack160_hercules/sorc/gdas.cd/build$ ctest -R test_gdasapp
Test project /work/noaa/da/rtreadon/git/global_workflow/spackstack160_hercules/sorc/gdas.cd/build
      Start 1340: test_gdasapp_util_coding_norms
 1/52 Test #1340: test_gdasapp_util_coding_norms ........................   Passed    4.28 sec

... 

      Start 1722: test_gdasapp_aero_gen_3dvar_yaml
52/52 Test #1722: test_gdasapp_aero_gen_3dvar_yaml ......................   Passed    0.29 sec

100% tests passed, 0 tests failed out of 52

Label Time Summary:
gdas-utils    =  20.64 sec*proc (9 tests)
script        =  20.64 sec*proc (9 tests)

Total Test time (real) = 1369.45 sec
```

Approve.

**Pull request merged**